.github/workflows/iroha2-dev.yml

```diff
@@ -122,3 +122,22 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           file: lcov.info
+
+  print-schema:
+    runs-on: [self-hosted, Linux] #ubuntu-latest
+    container: rust:1.52-buster
```

Comment:
Maybe it's time for 1.54?

Reply:
Next pr incoming. Fixing lints of clippy